app/views/layouts/_destroy.html.haml

```diff
@@ -0,0 +1,4 @@
+.error-summary{tabindex: '-1', role: 'group'}
+  %p.error-summary__message Are you sure you want to remove #{object.class}?
```

Comment:
this isn't what we discussed. Did you look at the link on how to do SASS/SCSS inheritance? 

i.e. the class should have a better representation name of whats happening but should still inherit the properties from error rather than rewriting them.

or is it not possible in this case?